cpp/arcticdb/storage/lmdb/lmdb_storage-inl.hpp

```diff
@@ -206,19 +206,16 @@ bool LmdbStorage::do_fast_delete() {
     // bool is probably not the best return type here but it does help prevent the insane boilerplate for
     // an additional function that checks whether this is supported (like the prefix matching)
     auto dtxn = ::lmdb::txn::begin(env());
-    ::lmdb::dbi default_dbi{0};
-
     foreach_key_type([&] (KeyType key_type) {
-        auto db_name = fmt::format("{}", key_type);
-        try {
-            ARCTICDB_SUBSAMPLE(LmdbStorageOpenDb, 0)
-            auto dbi = ::lmdb::dbi::open(dtxn, db_name.data());
-            ::lmdb::dbi_drop(dtxn, dbi, true);
-        }
-        catch (const ::lmdb::not_found_error &e) {
-            // Key type was not created, just skip.
+        if (key_type == KeyType::TOMBSTONE) {
```

Comment:
Oh wow that's not good, they are supposed to be unique. Not your problem to fix, but we should loop round soon and fix it.